ovn_k8s/common/util.py

```diff
@@ -0,0 +1,59 @@
+import json
+import random
+import subprocess
+
+import ovs.vlog
+from ovn_k8s.common import variables
+
+vlog = ovs.vlog.Vlog("util")
+
+
+def call_popen(cmd):
+    child = subprocess.Popen(cmd, stdout=subprocess.PIPE)
+    output = child.communicate()
+    if child.returncode:
+        raise RuntimeError("Fatal error executing %s" % (cmd))
+    if len(output) == 0 or output[0] is None:
+        output = ""
+    else:
+        output = output[0].strip()
+    return output
+
+
+def call_prog(prog, args_list):
+    cmd = [prog, "--timeout=5", "-vconsole:off"] + args_list
+    return call_popen(cmd)
+
+
+def ovs_vsctl(*args):
+    return call_prog("ovs-vsctl", list(args))
+
+
+def ovn_nbctl(*args):
```

Comment:
>It will be interesting to see how well using ovn-nbctl performs at scale. At some point we may have to switch to using the OVSDB protocol. There's a bunch of code we can steal borrow from the OpenStack plugin for this later.

Agreed. I am hoping to copy things from OpenStack plugin too.